components/navbar/navbar.module.scss

```diff
@@ -19,7 +20,42 @@
     padding: 14px 16px;
     text-decoration: none;
 }
-  
-.infoNavbar nav li a:hover {
+.infoNavbar nav li a:hover{
     background-color: rgba(255, 255, 255, 0.2);
 }
+.hamburger{
+    position: relative;
+    width: 35px;
+    height: 70%;
```

Comment:
Can we have height as auto?
because 70% height take a lot of space in Desktop view

Reply:
OK. I will make it correct.